geode-assembly/src/integrationTest/resources/dependency_classpath.txt

```diff
@@ -16,6 +15,7 @@ geode-redis-0.0.0.jar
 geode-serialization-0.0.0.jar
 geode-tcp-server-0.0.0.jar
 geode-wan-0.0.0.jar
+geode-management-0.0.0.jar
```

Comment:
why move this out of alphabetical order?

Reply:
Don't we touched this.. but we'll pay attention when we re-open this PR